Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs

```diff
@@ -130,10 +130,14 @@ public async Task Send(string modelCardId)
         }
       );
 
+      var activeUIDoc =
+        RevitContext.UIApplication?.ActiveUIDocument
+        ?? throw new SpeckleException("Unable to retrieve active UI document");
+
       List<ElementId> revitObjects = modelCard
         .SendFilter.NotNull()
         .GetObjectIds()
-        .Select(ElementIdHelper.Parse)
+        .Select(uid => ElementIdHelper.GetElementIdByUniqueId(activeUIDoc.Document, uid))
```

Comment:
i think there's some unecessary bouncing around here, from unique ids to element ids: in the root object builder, we can just use [GetElement(string)](https://www.revitapidocs.com/2019/b5a1473c-21c4-6c29-f1cf-26822c955260.htm) - where the string is the unique id.

Reply:
The helper function already do it but yes, simplified as you suggested